Modules/Config/MeleeSection.lua

```diff
@@ -235,11 +235,102 @@ function _Config:LoadMeleeSection()
+                get = function () return ExtendedCharacterStats.profile.melee.glance.display; end,
+                set = function (_, value)
+                    ExtendedCharacterStats.profile.melee.glance.display = value
+                    Stats:RebuildStatInfos()
+                end,
+            },
+            meleeGlanceGroup = {
+                type = "group",
+                order = 6,
+                inline = true,
+                name = function() return i18n("Melee Glance Values") end,
+                hidden = function()
+                    return (not ECS.IsClassic)
+                end,
+                args = {
+                    meleeGlance = {
+                        type = "toggle",
+                        order = 1,
+                        name = function() return i18n("Glancing Blow Chance") end,
+                        desc = function() return i18n("Shows/Hides the glancing chance against enemies on same level.") end,
+                        width = 1.5,
+                        disabled = function()
+                            return ((not ExtendedCharacterStats.profile.melee.display) or
+                                    (not ExtendedCharacterStats.profile.melee.glance.display))
+                        end,
+                        get = function () return ExtendedCharacterStats.profile.melee.glance.sameLevel.display; end,
+                        set = function (_, value)
+                            ExtendedCharacterStats.profile.melee.glance.sameLevel.display = value
+                            Stats:RebuildStatInfos()
+                        end,
+                    },
+                    meleeGlanceBoss = {
+                        type = "toggle",
+                        order = 2,
+                        name = function() return i18n("Glancing Blow Chance Boss") end,
+                        desc = function() return i18n("Shows/Hides the glancing chance against boss enemies (+3 Level).") end,
+                        width = 1.5,
+                        disabled = function()
+                            return ((not ExtendedCharacterStats.profile.melee.display) or
+                                    (not ExtendedCharacterStats.profile.melee.glance.display))
+                        end,
+                        get = function () return ExtendedCharacterStats.profile.melee.glance.bossLevel.display; end,
+                        set = function (_, value)
+                            ExtendedCharacterStats.profile.melee.glance.bossLevel.display = value
+                            Stats:RebuildStatInfos()
+                        end,
+                    },
+                    meleeGlanceDamage = {
+                        type = "toggle",
+                        order = 3,
+                        name = function() return i18n("Glancing Blow Damage") end,
+                        desc = function() return i18n("Shows/Hides the damage reduction against enemies on the same level.") end,
```

Comment:
The translation does not match with this. I like the wording though and think it should be the same for `meleeGlanceDamageBoss` a few lines below